components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/drop-elements.js

```diff
@@ -724,7 +731,16 @@ define(['require', 'log', 'lodash', 'jquery', 'partition', 'stream', 'query', 'f
             if (isCodeToDesignMode) {
                 name = functionName;
             } else {
-                name = self.formBuilder.DefineFunction(i);
+                //add the new function element to function list
+				var functionOptions = {};
```

Comment:
format your code

Reply:
Fixed in 97aff2523a4468759c33ce94e34dd1c88ea092af